src/components/pages/maintenance/maintenance.js

```diff
@@ -119,6 +122,12 @@ class MaintenancePage extends Component {
         const ruleId = updatedAlarms[0].Rule.Id;
         this.props.actions.updateAlarmsStatus({ alarmIds, status, ruleId });
       }
+      setTimeout(() => {
+        this.setState({ showIndicator: false });
```

Comment:
Why hide the indicator on a timeout?

Reply:
design requirement to show at least one fully loading animation